src/main/resources/site/pages/rss/rss.js

```diff
@@ -81,20 +81,20 @@ exports.get = function (req) {
 
     }
 
-    var params = {
+    const params = {
         content: content,
         posts: posts,
         pageUrl: pageUrl,
         site: site
     };
 
-    var view = resolve('rss.xsl');
-    //var copy = resolve('copy-of.xsl');
+    const view = resolve('rss.xsl');
+    //const copy = resolve('copy-of.xsl');
 
-    var body = xslt.render(view, params);
```

Comment:
xslt 😱 🙀 Will have to look at this in the future some time.

Reply:
Agree!!!!! I thought I'd improve these other js files (other than default.js) in separate PRs, this PR only focuses on `var`.